test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/ControllerActionInvokerTest.cs

```diff
@@ -33,43 +33,43 @@ public class ControllerActionInvokerTest : CommonResourceInvokerTest
     {
         #region Diagnostics
 
-        [Fact]
-        public async Task Invoke_Success_LogsCorrectValues()
```

Comment:
Temporarily disabled them. Wanted to get some initial feedback to see if we want to follow the before/after approach and then modify these 2 commented out tests.